Java/HeapSort.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Arrays;
```

Comment:
Mention Time Complexity

Reply:
Please check the updated PR.

Reply:
@manisaiprasad anything else do i need to update?